auto-generated-sdk/docs/ComponentSummary.md

```diff
@@ -6,6 +6,7 @@ Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
 **name** | **str** | Component name. | [optional] 
 **category** | **str** | Component category. | [optional] 
+**type** | **str** | Component type. | [optional] 
```

Comment:
Can you please revert all the files except for the example files?

Reply:
done https://github.com/factset/analyticsapi-engines-python-sdk/pull/62/commits/fe9035a13aa3b2b4766f730ad376fdfa3b244a51